src/dags/importscripts/import_sport.py

```diff
@@ -0,0 +1,163 @@
+import csv
+from hashlib import blake2s
+import json
+import logging
+from typing import List, Tuple, TypedDict, Generator, Type
+
+
+# define logger for output to console
+logger = logging.getLogger(__name__)
+
+
+def get_dialect(file: str) -> Type:
```

Comment:
Some type annotations can be a bit more specific. You're returning a `Dialect` type here, hence `-> Type[Dialect]`

Reply:
Aangepast